Makefile

```diff
@@ -658,7 +658,9 @@ vet:
 
 # Generate code
 generate: $(BINDIR)/controller-gen
-	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+	$(CONTAINERIZED) $(CALICO_BUILD) sh -c '$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."'
+	# Make sure the generated code won't cause a static-checks failure.
+	$(MAKE) fix
```

Comment:
It's a bit surprising that this has to be run. I am okay with it, just surprised that generated code isn't formatted right.

Reply:
we do this all over the place at the moment

I think the formatting adds in some extra build directives that I guess the generation code has not yet been updated to add.